src/core/constants.js

```diff
@@ -1342,4 +1342,30 @@ export const HALF_FLOAT = 'half-float';
  * @property {RGBA} RGBA
  * @final
  */
-export const RGBA = 'rgba';
\ No newline at end of file
+export const RGBA = 'rgba';
+
+/**
+ * The `splineEnds` mode where splines curve through
+ * their first and last points.
+ * @typedef {unique symbol} P2D
```

Comment:
Change `* @typedef {unique symbol} P2D` to `* @typedef {unique symbol} SHOW`? Same for `HIDE` and `END`.

Reply:
thanks, I'll update those!